pubspec.lock

```diff
@@ -816,10 +816,18 @@ packages:
     dependency: "direct main"
     description:
       name: flutter_inappwebview
-      sha256: d198297060d116b94048301ee6749cd2e7d03c1f2689783f52d210a6b7aba350
+      sha256: "786b124c944f4914670868c9775c90600a4d4674497f617190c0d676019e337b"
```

Comment:
we need it?

Reply:
we don't really need it, but because I have updated it so I think we can keep it :). wdyt ?

Reply:
dont keep what we don't need :)))